src/new-components/stack/stack.tsx

```diff
@@ -1,29 +1,46 @@
 import * as React from 'react'
 import flattenChildren from 'react-keyed-flatten-children'
 import { polymorphicComponent } from '../../utils/polymorphism'
-import { getClassNames } from '../responsive-props'
+import { getClassNames, mapResponsiveProp } from '../responsive-props'
 import { Box } from '../box'
 import { Divider } from '../divider'
 
 import type { ResponsiveProp } from '../responsive-props'
 import type { Space } from '../common-types'
-import type { ReusableBoxProps } from '../box'
+import type { BoxProps, ReusableBoxProps } from '../box'
 import type { DividerWeight } from '../divider'
 
 import styles from './stack.module.css'
 
 interface StackProps extends ReusableBoxProps {
+    /** Space between items */
     space?: ResponsiveProp<Space>
+    /** Align items horizontally */
+    align?: ResponsiveProp<'left' | 'center' | 'right'>
```

Comment:
Other components have `alignX` and `alignY` props... does it make sense to rename this to `alignX`? 🤔

Reply:
We don't use `alignX`. We're pretty much aligned (no pun intended) in that `align` is for horizontal alignment and `alignY` for vertical alignment.

Reply:
I'll go ahead and merge, but if something actionable comes up from this discussion, I can always include the changes in subsequent PRs. So let me know.